frontend/packages/shared/src/version-control/CloneModal.tsx

```diff
@@ -88,7 +85,11 @@ export function CloneModal(props: ICloneModalProps) {
         )}
         <>
           <div className={classes.blackText}>{t('sync_header.clone_https')}</div>
-          <TextField id='repository-url-form' value={repositoryGitPath(org, app)} readOnly />
+          <TextField
+            id='repository-url-form'
+            value={window.location.origin.toString() + repositoryGitPath(org, app)}
```

Comment:
What do you think about reusing your variable on line 23? :) 
```suggestion
            value={gitUrl}
```